### DISCUSSION TITLE
Is there synchronized BatchNormalization for JAX or torch backends?

### DISCUSSION THREAD

**Discussion by caiuspetronius** [keras-team/keras#20354] [General]
Currently, Keras only accepts synchronized = True in the BatchNormalization() while using the TF backend. How do I achieve the same with the JAX or torch backend?
